Composer/packages/client/src/pages/publish/BotStatusList.tsx

```diff
@@ -32,25 +33,35 @@ const copiedCalloutStyles = {
 };
 
 type SkillManifestUrlFieldProps = {
-  url: string;
+  urls: string[];
 };
 
-const SkillManifestUrlField = ({ url }: SkillManifestUrlFieldProps) => {
-  const { isCopiedToClipboard, copyTextToClipboard, resetIsCopiedToClipboard } = useCopyToClipboard(url);
+const SkillManifestUrlField = ({ urls }: SkillManifestUrlFieldProps) => {
+  const { isCopiedToClipboard, copyTextToClipboard, resetIsCopiedToClipboard } = useCopyToClipboard(urls[0]);
 
   const calloutTarget = useRef<HTMLElement>();
+  const clipUrl = (url) => url.replace('azurewebsites.net/manifests/', 'azureweb...');
   return (
     <Fragment>
-      <ActionButton
-        className="skill-manifest-copy-button"
-        title={url}
-        onClick={(e) => {
-          calloutTarget.current = e.target as HTMLElement;
-          copyTextToClipboard();
-        }}
+      <TooltipHost
+        content={urls.map((url) => (
+          <div key={url} style={{ display: 'flex' }}>
+            {clipUrl(url)}
+            <IconButton iconProps={{ iconName: 'copy' }} onClick={() => navigator.clipboard.writeText(url)} />
+          </div>
+        ))}
       >
-        {formatMessage('Copy Skill Manifest URL')}
-      </ActionButton>
+        <ActionButton
+          className="skill-manifest-copy-button"
```

Comment:
why is this needed? if for testing, use data-testid

Reply:
remove it